pkgs/development/python-modules/skein/skeinrepo.nix

```diff
@@ -1,9 +1,9 @@
 { autoPatchelfHook, lib, maven, stdenv, src, version }:
 
 stdenv.mkDerivation rec {
-  name = "skein-${version}-maven-repo";
+  pname = "skein-maven-repo";
 
-  inherit src;
```

Comment:
this one shhould probably be undone.

It's a FOD, so the derivation name changing will influence the sha256

Reply:
could the sha256 in this file be recomputed? or what do you mean by FDO? foreign object damage?

Reply:
Oh wait, sorry. I'm mixing things up again. The content should be the same. It's just that the build will have to be re-computed for nix to be happy. This is fine.

Reply:
It would only affect the sha if the `outputHashMode` was flat. But it's recursive.